### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by FinalAngel**
https://requires.io/github/aldryn/aldryn-newsblog/requirements/?branch=master

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     'django-reversion>=1.8.2,<1.9',
     'django-sortedm2m',
     'django-taggit',
-    'django>=1.6,<1.8',
+    'django>=1.6,<1.8', # rq.filter: 1.6,<1.8
```
Comment (Chive) [inline setup.py]: Is this for requires.io? I am not a fan of cluttering files with comments so some 3rd party services can work with it..

File: test_requirements/base.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@ django-appdata>=0.1.4
 django-classy-tags
 # django-cms>=3.0.12
 django-filer
-django-mptt<0.7.0
+django-mptt<0.7.4
```
Comment (Chive) [inline test_requirements/base.txt]: **Not** sure if this upgrade is save.

File: setup.py (hunk 1 above)
Comment (intellectronica) [inline setup.py]: I agree with @Chive.

File: test_requirements/base.txt (hunk 2 above)
Comment (yakky) [inline test_requirements/base.txt]: I know that 0.7.0/0.7.1 broke the CMS, but apparently 0.7.3+ are safe and it's the only way to support Django 1.8. Test suite runs ok, so I'd accept it (mptt breakages are usually very obvious and final)

File: setup.py (hunk 1 above)
Comment (intellectronica) [inline setup.py]: Also, flake8 doesn't like this because PEP8 dictates two spaces between code and comment.

**Comment by FinalAngel** [comment]
alright I will close this than as it breaks tests and wont fit our guidelines, the goal was to fix requirements.io

**Pull request closed without merging**